--- old\CMakeLists.txt	Fri Mar 23 22:33:36 2018
+++ new\CMakeLists.txt	Thu Dec 20 16:47:51 2018
@@ -913,6 +913,14 @@
       DEBUG_POSTFIX "${MSVC_TOOLSET}-mt-sgd-${ZMQ_VERSION_MAJOR}_${ZMQ_VERSION_MINOR}_${ZMQ_VERSION_PATCH}"
       COMPILE_FLAGS "/DZMQ_STATIC"
       OUTPUT_NAME "libzmq")
+
+	foreach (flag_var
+             CMAKE_CXX_FLAGS CMAKE_CXX_FLAGS_DEBUG CMAKE_CXX_FLAGS_RELEASE
+             CMAKE_CXX_FLAGS_MINSIZEREL CMAKE_CXX_FLAGS_RELWITHDEBINFO)
+
+        string(REPLACE "/MD" "-MT" ${flag_var} "${${flag_var}}")
+	endforeach()
+
   endif()
 else ()
   # avoid building everything twice for shared + static
